tests/encoder.cpp

```diff
@@ -100,13 +100,13 @@ class EncoderTest : public ::testing::Test {
 };
 
 void AssertReadRaw(std::stringstream &sstr, const std::string &expected) {
-  char tmp[expected.size()];
-  size_t read = sstr.readsome(tmp, expected.size());
+  std::vector<char> tmp(expected.size());
```

Comment:
Same question as before, what was the warning here?

Reply:
Yea, the same as above, any alternative to the vector? But I think it's pretty unimportant in general...

Reply:
Yeah, it's not important just was curious. Vector should be fine or:
`auto tmp = std::make_unique<char[]>(expected.size())`?

Reply:
Let's leave vector then :smile: